tools/setup/emoji/build_emoji

```diff
@@ -92,7 +92,8 @@ def main() -> None:
     if not os.path.exists(success_stamp):
         print("Dumping emojis ...")
         dump_emojis(source_emoji_dump)
-        open(success_stamp, 'w').close()
+        with open(success_stamp, 'w')as f:
```

Comment:
This whitespace is off (and maybe invalid?)

Reply:
Yeah, weird I'm surprised I didn't get lint errors on this.